src/navigations/HomeBottomTabs.js

```diff
@@ -106,24 +106,9 @@ function HomeBottomTabs({navigation}) {
     // Should the initial notification be popped automatically
     // default: true
     popInitialNotification: true,
-    requestPermissions: true
+    requestPermissions: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `requestPermissions: true` to `requestPermissions: false` might cause issues if the app relies on this configuration to request permissions for notifications. If the permission is not requested elsewhere in the code, the app might fail to receive notifications.
```diff
-    requestPermissions: false
+    requestPermissions: true
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->